mwdb/model/migrations/versions/d7725a4e500c_tag_model_flattening.py

```diff
@@ -1,7 +1,7 @@
 """Tag model flattening
 
 Revision ID: d7725a4e500c
-Revises: b78affca7273
+Revises: 1b49fefdfcb0
```

Comment:
This change is not connected to jwt tokens, but I saw inconsistency, so I've changed it